doc/guides/debugging/host-tools.rst

```diff
@@ -88,3 +97,6 @@ Check if your SoC is listed in `pyOCD Supported Devices`_.
 
 .. _pyOCD Supported Devices:
    https://github.com/mbedmicro/pyOCD/tree/master/pyocd/target/builtin
+
+.. _OpenOCD Windows:
+    http://gnutoolchains.com/arm-eabi/openocd/
```

Comment:
Why here? What about the GNU MCU Eclipse mentioned in the "Getting OpenOCD" page?

Reply:
Seems this may not be the best way to get up to date openocd binaries (for windows at least). For windows, it redirects to http://www.freddiechopin.info/en/download/category/4-openocd with latest build having: 'build date: 2017-01-23'. (which is older than the oldest binary in http://gnutoolchains.com/arm-eabi/openocd/)